nautobot/dcim/elevations.py

```diff
@@ -9,9 +13,9 @@
 from .constants import RACK_ELEVATION_BORDER_WIDTH
 
 
-class RackElevationSVG:
+class RackElevationGraphicalOutput:
```

Comment:
This isn't a "public" API (not exposed in `nautobot.apps` namespace) but I still feel a bit nervous renaming it without providing some sort of backwards-compatibility shim.